java/ql/test/ext/TopJdkApis/TopJdkApisTest.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Test for top JDK APIs
+ * @description Provides a test case to check that a manual models exists for each each of the top JDK APIs.
+ * @id java/top-jdk-apis-test
+ */
+
+import java
+import semmle.code.java.dataflow.ExternalFlow
+import TopJdkApis
+
+from string apiName, TopJdkApi topApi
+where
+  // top jdk api names for which there is no callable
+  topJdkApiName(apiName) and
+  not hasCallable(apiName)
+  or
+  // top jdk api names for which there isn't a manual model
+  not topApi.hasManualMadModel() and
+  apiName =
+    topApi.asCallable().getDeclaringType().getPackage() + "." +
+      topApi.asCallable().getDeclaringType().getSourceDeclaration() + "#" +
+      topApi.asCallable().getName() + paramsString(topApi.asCallable())
+select apiName order by apiName
```

Comment:
Add a message column that states which of the two cases we're in, so we can see in the test output whether there's no callable, or whether there isn't a manual model.

Reply:
Updated in https://github.com/github/codeql/pull/11572/commits/640b450c4783aed2061b58fcb850d79cd414b40c.